test-cypress/integration/cluster/cluster-configuration.spec.js

```diff
@@ -0,0 +1,277 @@
+                ClusterPageSteps.previewClusterConfig();
+                ClusterConfigurationSteps.selectTab(2);
+                // I expect to see
+                ClusterConfigurationSteps.getMultiRegionHeader().should('contain.text', 'Primary cluster');
+                ClusterConfigurationSteps.getAddTagButton().should('be.visible').and('not.be.disabled');
+                ClusterConfigurationSteps.getEnableSecondaryModeButton().should('be.visible').and('not.be.disabled');
+                ClusterConfigurationSteps.getTagsTable().should('not.exist');
+
+                ClusterStubs.stubAddTag(tagName);
+                ClusterConfigurationSteps.clickAddTagButton();
+                ClusterConfigurationSteps.typeTag(tagName);
+                ClusterConfigurationSteps.clickSubmitTagButton();
+                cy.wait('@add-tag').then((interception) => {
+                    expect(interception.request.body).to.deep.equal({tag: tagName});
+                });
+
+                ClusterStubs.stubClusterGroupStatusWithTag();
+                cy.wait('@3-nodes-cluster-group-status-tag').then(() => {
+                    // Assert the tags table contains the expected tag
+                    ClusterConfigurationSteps.getTagsTable().should('be.visible');
+                    ClusterConfigurationSteps.getTagsTableRows().should('contain.text', tagName);
+                });
+                // And expect success message to be displayed.
+                ApplicationSteps.getSuccessNotifications().contains(`Successfully added to cluster primary identifier tag: ${tagName}`);
+
+                //When I delete the tag
+                ClusterConfigurationSteps.clickDeleteTagButton();
+                // I expect to see deleting confirmation dialog.
+                ModalDialogSteps.getDialogHeader().should('contain', `Delete identifier tag ${tagName}`);
+                ModalDialogSteps.getDialogBody().should('contain', 'Deleting identifier tag would stop any secondary cluster identified with it from pulling updates.');
+
+                // When I confirm
+                ClusterStubs.stubDeleteTag(tagName);
+                ModalDialogSteps.getConfirmButton().click();
+                cy.wait('@delete-tag').then((interception) => {
+                    expect(interception.request.body).to.deep.equal({tag: tagName});
+                });
+            });
+        });
+
+        context('Secondary cluster', () => {
+            it('should be able to switch modes', () => {
+                ClusterStubs.stubEnableSecondaryMode();
+                const rpcAddress = 'node-name:7300';
+                const tag = 'us-central';
+
+                // Given I have opened the cluster management page
+                ClusterPageSteps.visit();
+                // When I click on edit properties and open Nodes tab
+                ClusterPageSteps.previewClusterConfig();
+                ClusterConfigurationSteps.selectTab(2);
```

Comment:
Maby just create a method `selectMultiRegionConfigTab` or something.

Reply:
created, for node tab also